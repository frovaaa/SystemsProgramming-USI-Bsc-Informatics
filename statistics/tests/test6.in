0 40 40 40
