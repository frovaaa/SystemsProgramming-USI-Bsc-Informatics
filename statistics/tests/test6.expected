0.000000 30.000000 40.000000
